report_purchase_nrq/report/purchase_report_template.xml

```diff
@@ -53,7 +53,16 @@
                                 />
                             </td>
                             <td class="text-right">
-                                <span t-field="line.price_unit" />
+                                <t t-set="price_unit" t-value="line.price_unit" />
+                                <t t-if="round(price_unit) == price_unit">
+                                    <span
+                                        t-esc="float('%.2f'%(price_unit))"
+                                        t-options='{"widget": "monetary", "display_currency": doc.currency_id, "precision":2}'
```

Comment:
Why should this this `t-options` be here?

Reply:
Now t-esc is float, so if t-option not be here, currency symbol and comma(,) will not appear.

Reply:
> Now t-esc is float, so if t-option not be here, currency symbol and comma(,) will not appear.

It makes the field presentation inconsistent whether there is something below decimal or not. `price_unit` is a float field in the first place (i.e. currently no currency symbol shows).

Get rid of `t-options`, and try replacing
```
t-esc="float('%.2f'%(price_unit))"
```
with
```
t-esc="'%:,.0f'%(price_unit)"
```
to see if it works?

Please also double-check the assumption - my understanding is that we are supposed to change the decimal precision setting of 'Product Price' from `0` to `2`.

Reply:
@yostashiro 
It not works.
If you can, please check how it works with comma.

Reply:
@TomohiroAbe-Quartile Take a look here: https://github.com/qrtl/rpl-private/pull/209/commits/3d75df04a555f4de1f58888483503901fea34270#diff-0e1826db127901de699b3b33bddebec4ece23baf5430c9e3a241dd65c10781f1R36-R37

Reply:
Thank you, it works as expected. I will learn why this line does working.